src/App.js

```diff
@@ -14,11 +14,66 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+  };
+
+  addGood = (good) => {
+    this.setState({
+      selectedGood: good,
+      isButtonHiden: true,
+    });
+  }
+
+  removeGood = () => {
+    this.setState({
+      selectedGood: 'No goods',
+      isButtonHiden: true,
+    });
+  }
+
+  render() {
+    const { selectedGood, isButtonHiden } = this.state;
+
+    return (
+      <div className="App">
+        <h1>
+          {`Selected good: - ${selectedGood} selected `}
+          <button
+            type="button"
+            hidden={isButtonHiden}
```

Comment:
- use conditional rendering to hide something
- instead of isButtonHiden just check if good selected